parser/parser.y

```diff
@@ -560,6 +561,15 @@ AlterSpecification:
 KeyOrIndex:
 	"KEY"|"INDEX"
 
+ColumnOptColumnName:
+	"COLUMN"
+|	"COLUMN" "COLUMN"
```

Comment:
"Alter table t drop column int;" is a valid sql statement but "int" is not an identifier in parser.y. @cznic 
"Alter table t drop column column;" is also a valid sql statement. @siddontang